code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -895,6 +948,7 @@
 	color = "#102000"
 	adj_drowsy = -6
 	adj_temp = -5
+	taste_tag = list(TASTE_SOUR,TASTE_BITTER,TASTE_SWEET,TASTE_STRONG, TASTE_LIGHT, TASTE_BUBBLY, TASTE_SPICY, TASTE_SALTY)
```

Comment:
Is this intended?

Reply:
yes, doctor gib have over 48 flavors its said. altough, its set as a "junk food" so it shouldn't work for insight

Reply:
A delicious blend of 42 different flavours

Reply:
desc says. A delicious blend of 42 different flavours

 so i followed it

Reply:
Based

Reply:
This might be a bit unbalanced as these can be bought at any soda machine.

Reply:
this gave a idea how about making the can price 1k? might be over killing it. but still how about it